orebfuscator-plugin/src/main/java/net/imprex/orebfuscator/obfuscation/DeobfuscationListener.java

```diff
@@ -40,49 +41,49 @@ private DeobfuscationListener(Orebfuscator orebfuscator, DeobfuscationWorker deo
 		this.deobfuscationWorker = deobfuscationWorker;
 	}
 
-	@EventHandler
+	@EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
```

Comment:
Basically ok, but I would prefer `EventPriority.HIGH` so that other plugins still have the option to ignore Orebfusactor for whatever reason

Reply:
> Basically ok, but I would prefer `EventPriority.HIGH` so that other plugins still have the option to ignore Orebfusactor for whatever reason
What do you mean by ignoring Orebfuscator?

Reply:
I would prefer `EventPriority.HIGH` because if a plugin for whatever reason wants to edit a event and hide the change from other plugins such a Orebfuscator their would still be `EventPriority.HIGHEST` and `EventPriority.MONITOR`. I just don't see the point in using anything different then `EventPriority.NORMAL` or `EventPriority.HIGH` here since that leaves other plugins with plenty of options the be execute before or after Orebfuscator if they desire to.